WordPress/src/main/java/org/wordpress/android/ui/reader/utils/ReaderImageScanner.java

```diff
@@ -14,7 +13,7 @@
     private final boolean mContentContainsImages;
 
     private static final Pattern IMG_TAG_PATTERN = Pattern.compile(
-            "<img(\\s+.*?) (?:src\\s*=\\s*(?:'|\") (.*?) (?:'|\")) (.*?)>",
+            ".*(<img\\s+.*src\\s*=\\s*\"([^\"]+)\".*>).*",
```

Comment:
Are we sure that only double-quotes will define the `src` attribute? The current pattern seems to account for single-quotes as well.

Reply:
I checked and every image used double quotes (and, strangely, every `iframe` used single quotes). My assumption is our backend's HTML tidier automatically provides this formatting.